README.md

```diff
@@ -7,9 +7,9 @@ Ballerina XSLT Library
   [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
   [![codecov](https://codecov.io/gh/ballerina-platform/module-ballerina-xslt/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-xslt)
 
-The XSLT library is one of the standard library modules of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+The XSLT library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
 
-This module provides a function to transform the XML content to another XML/HTML/plain text using XSL transformations.
+This package provides a function to transform the XML content to another XML/HTML/plain text using XSL transformations.
 
 For more information go to [The XSLT Module](https://ballerina.io/swan-lake/learn/api-docs/ballerina/xslt/).
```

Comment:
```suggestion
For more information, go to the [`xslt` module](https://ballerina.io/swan-lake/learn/api-docs/ballerina/xslt/).
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-xslt/pull/138/commits/18cfd06cc69a76b0cb2a6d6bff3e7c847b6c8304